cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1704,15 +1717,28 @@ class TestEditSplitModule(ItemTest):
     def setUp(self):
         super(TestEditSplitModule, self).setUp()
         self.user = UserFactory()
+
+        self.first_user_partition_group_1 = Group("101", 'alpha')
+        self.first_user_partition_group_2 = Group("102", 'beta')
+        self.first_user_partition = UserPartition(
+            100, 'first_partition', 'First Partition',
+            [self.first_user_partition_group_1, self.first_user_partition_group_2]
+        )
+
+        self.second_user_partition_group_1 = Group("101", 'Group 1')
```

Comment:
Note to reviewers: there is a test point in this class that purposefully wanted the group IDs to be non-unique.